crawl-ref/source/dat/descript/monsters.txt

```diff
@@ -938,11 +938,11 @@ dire elephant
 A huge pachyderm, wilder and fiercer than normal elephants, with a very thick
 hide, powerful legs, and a memory to die for.
 %%%%
-disaster prism
+disaster emanation
 
 A spark created by the meeting of adversaries, bound and shaped by powerful magic.
-It will explode with dreadful force not long after the magic that enabled its creation
-expires.
+It will explode with dreadful force not long after the magic that enabled its
+creation expires.
```

Comment:
This was in a different pull request and isn't related to your spell, they should be separated (aka remove them from this pull request) to make it easier to merge.